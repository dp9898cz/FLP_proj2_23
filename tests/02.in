S S R a
s a B R
S c S R
S a A a
A a A R
A b B b
A c A R
B b B L
B a B b
B c F c
caaabaa