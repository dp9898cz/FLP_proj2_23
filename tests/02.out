Scaaabaa
cSaaabaa
cAaaabaa
caAaabaa
caaAabaa
caaaAbaa
caaaBbaa
caaBabaa
caaBbbaa
caBabbaa
caBbbbaa
cBabbbaa
cBbbbbaa
Bcbbbbaa
Fcbbbbaa